include/glow/Importer/Caffe2ModelLoader.h

```diff
@@ -60,8 +60,16 @@ class Caffe2ModelLoader
   /// in the network.
   llvm::Error loadOperator(const caffe2::OperatorDef &op);
 
-  /// Reads a network (weights or structure) from the serialized protocol buffer
-  /// file.
+  // Load the Conv or ConvRelu operators.
+  llvm::Error loadConv(const caffe2::OperatorDef &op,
```

Comment:
Looks like this and `loadConvQuantized()` are essentially just helpers for the Caffe2ModelLoader -- perhaps we can make them just static inside Caffe2ModelLoader.cpp and not declare them here?